mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -377,3 +383,13 @@ def _id_counter(self):
         while True:
             yield i
             i += 1
+
+    def set_verbose_logging(self, verbose: bool = False):
```

Comment:
Need to consider python 2 compatibility.

Reply:
Can you be more specific?
What's not py2 compatible here?

Reply:
Could found `SyntaxError: invalid syntax` in the [py2 CI logs](https://travis-ci.org/github/google/mobly/jobs/735993600) below:    
```
E     File "/home/travis/build/google/mobly/mobly/controllers/android_device_lib/jsonrpc_client_base.py", line 387
E       def set_verbose_logging(self, verbose: bool = False):
E                                            ^
E   SyntaxError: invalid syntax
```
I think maybe need to use this kind of approach to workaround the python 2 compatibility issue:   
```
    def set_snippet_client_verbose_logging(self, verbose = False):
    # type: (bool) -> None
```

By the way, [CI also fail in py3](https://travis-ci.org/github/google/mobly/jobs/735993600) due to "'NoneType' object is not subscriptable":
```
>                          response[:MAX_RPC_RETURN_VALUE_LENGTH])
E                          TypeError: 'NoneType' object is not subscriptable
```

Reply:
I see. two things:
1. In this PR, let's not use py type
2. Eric, can you fix the CI asap :)

Reply:
I created [PR 684](https://github.com/google/mobly/pull/684) with:
* all PASS CI
* unit test cases
* as-is behavior
* configurable `jsonrpc_client_base.SNIPPET_RESPONSE_LOGGING_LENGTH_LIMITATION`

Reply:
I modify it based on my version with
- Let length in class variable and configurable
- More unit tests